app/(dashboard)/mailboxes/[mailbox_slug]/sessions/sessionsList.tsx

```
@@ -2,6 +2,7 @@
 
 import { formatDistanceToNow } from "date-fns";
 import { Calendar, Eye, Loader2 } from "lucide-react";
+import { SessionsListSkeleton } from "@/components/skeletons/SessionsListSkeleton";
 import { useRouter } from "next/navigation";
```

ESLint

[error] Delete `SessionsListSkeleton·}·from·"@/components/skeletons/SessionsListSkeleton";⏎import·{·`

(prettier/prettier)